src/parsers/Uniref_mapping.py

```diff
@@ -0,0 +1,467 @@
+## wget ftp://ftp.uniprot.org/pub/databases/uniprot/uniref/uniref100/uniref100.xml.gz is too big
```

Comment:
can you remove this file from the PR so we can just concentrate on the uniprot parser?